delphi-checks/src/test/java/au/com/integradev/delphi/checks/MixedNamesCheckTest.java

```diff
@@ -23,8 +23,30 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
+import org.sonar.plugins.communitydelphi.api.check.DelphiCheck;
 
 class MixedNamesCheckTest {
+  private static DelphiCheck createCheck() {
+    MixedNamesCheck check = new MixedNamesCheck();
+    check.excludedNames = "b_IgnoreBool";
+    return check;
+  }
+
+  @Test
+  void testNameInExcludedListShouldNotAddIssue() {
+    CheckVerifier.newVerifier()
+        .withCheck(createCheck())
+        .onFile(
+            new DelphiTestUnitBuilder()
+                .appendImpl("procedure Test;")
+                .appendImpl("var")
+                .appendImpl("  b_IGNOREBOOL: Boolean;")
+                .appendImpl("begin")
+                .appendImpl("  b_IgnoreBool := True;")
+                .appendImpl("end;"))
+        .verifyNoIssues();
```

Comment:
Could you add a test that asserts that the exclusion check is not case-sensitive? e.g. having the excluded name be `MyIdentifierName`, but the declaration be `MYIDENTIFIERNAME` and the usage be `myidentifiername`.